images/cilium/Dockerfile

```diff
@@ -17,9 +19,10 @@ RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium --mount=
   HOST_CC=aarch64-linux-gnu-gcc HOST_STRIP=aarch64-linux-gnu-strip \
   SKIP_DOCS=true DESTDIR=/out/linux/arm64 clean-container build-container install-container
 
-FROM docker.io/errordeveloper/cilium-runtime:201594d798c0dafdc5969e81ca21fa2761103b7f-dev
-LABEL maintainer="maintainer@cilium.io"
+FROM ${CILIUM_RUNTIME_IMAGE}
 ARG TARGETPLATFORM
+LABEL maintainer="maintainer@cilium.io"
```

Comment:
I think this is also missing the `ARG CILIUM_SHA=""`, right?

Reply:
That was only needed due to how image GC worked on Jenkins.

Reply:
To clarify, do we want the Cilium SHA as a label for subsequent introspection?

Reply:
It's actually being used as image tag for the agent image, as well `operator` and `hubble-relay`.